submissions/Temu4/html_css_popup/style.css

```diff
@@ -0,0 +1,254 @@
+body {
+  font-family: sans-serif;
+  font-size: 13px;
+}
+
+.menu-bar {
+  display: flex;
+  width: 300px;
+  float: right;
+  vertical-align: middle;
+}
+
+/**  Links **/
+a {
+  width: 20%;
+  margin: auto;
+  color: rgb(22, 22, 24);
+  text-align: center;
+  outline: none;
```

Comment:
All interactive elements should have :focus state

Reply:
Dont use styling by tag

Reply:
Also, all interactive elements should have `:active` state.